docs_src/examples/Ch-ExamplesUsbDeviceService.md

```diff
@@ -0,0 +1,367 @@
+                    	"Path": "/dev/video6",
+ 			                "AutoStreaming": "false"
+                     }
+                  }
+               }
+            }
+   ]'
+   ```
+
+   Example Output: 
+   ```bash
+   [{"apiVersion":"v2","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
+   ```
+
+## Start Video Streaming
+Unless the device service is configured to stream video from the camera automatically, a `StartStreaming` command must be sent to the device service.
+
+There are two types of options:
+- The options start with `Input` prefix are used for the camera, such as specifying the image size and pixel format.
+- The options start with `Output` prefix are used for the output video, such as specifying aspect ratio and quality.
+
+These options can be passed in through Object value when calling StartStreaming.
+
+Query parameter:
+- `device name`: The name of the camera
+
+For example:
+```shell
+curl -X PUT -d '{
+    "StartStreaming": {
+      "InputImageSize": "640x480",
+      "OutputVideoQuality": "5"
+    }
+}' http://localhost:59882/api/v2/device/name/<device name>/StartStreaming
+```
+
+Supported Input options:
+- `InputFps`: Ignore original timestamps and instead generate timestamps assuming constant frame rate fps. (default - same as source)
+- `InputImageSize`: Specifies the image size of the camera. The format is `wxh`, for example "640x480". (default - automatically selected by FFmpeg)
+- `InputPixelFormat`: Set the preferred pixel format (for raw video). (default - automatically selected by FFmpeg)
+
+Supported Output options:
+- `OutputFrames`: Set the number of video frames to output. (default - no limitation on frames)
+- `OutputFps`: Duplicate or drop input frames to achieve constant output frame rate fps. (default - same as InputFps)
+- `OutputImageSize`: Performs image rescaling. The format is `wxh`, for example "640x480". (default - same as InputImageSize)
+- `OutputAspect`: Set the video display aspect ratio specified by aspect. For example "4:3", "16:9". (default - same as source)
+- `OutputVideoCodec`: Set the video codec. For example "mpeg4", "h264". (default - mpeg4)
+- `OutputVideoQuality`: Use fixed video quality level. Range is a integer number between 1 to 31, with 31 being the worst quality. (default - dynamically set by FFmpeg)
+
+
+### Determine Stream Uri of Camera
+The device service provides a way to determine the stream URI of a camera.
+
+Query parameter:
+- `device name`: The name of the camera
+
+```bash
+curl -s http://localhost:59882/api/v2/device/name/<device name>/StreamURI | jq -r '"StreamURI: " + '.event.readings[].value''
+```
+
+The response to the above call should look similar to the following:
+
+```
+StreamURI: rtsp://localhost:8554/stream/NexiGo_N930AF_FHD_Webcam__NexiG-20201217010
+```
+
+### Stream the RTSP stream. 
+
+   ffplay can be used to stream. The command follows this format: 
+   
+   `ffplay -rtsp_transport tcp rtsp://<IP address>:<port>/<streamname>`.
```

Comment:
Is this why ffmpeg package is installed in the beginning? I suggest using something simpler like `mplayer` so ffmpeg package and all its dependencies aren't needed.

```
mplayer rtsp://<IP address>:<port>/<streamname>
```